examples/whisper/whisper.py

```diff
@@ -7,145 +7,152 @@
-
-new_data = None
-keep_data = None
-while running and not disconnected:
-    if audio_sink is None:
-        time.sleep(0.01)
-        continue
-    success, input_data = audio_sink.read()
-    if not success:
-        time.sleep(0.01)
-        continue
-    if new_data is None:
-        new_data = input_data
-    else:
-        new_data = np.concatenate([new_data, input_data])
-    if not running or disconnected:
-        break
-    if new_data.shape[0] < n_samples_step:
-        time.sleep(0.01)
-        continue
-    if new_data.shape[0] > 2*n_samples_step:
-        print("リアルタイムで処理できる性能が無いため処理ができませんでした")
         new_data = None
         keep_data = None
-        continue
-    n_samples_new = new_data.shape[0]
+        while self.running and not self.disconnected:
+            if self.audio_sink is None:
+                time.sleep(0.01)
+                continue
+            success, input_data = self.audio_sink.read()
+            if not success:
+                time.sleep(0.01)
+                continue
+            if new_data is None:
+                new_data = input_data
+            else:
+                new_data = np.concatenate([new_data, input_data])
+
+            if not self.running or self.disconnected:
+                break
+            if new_data.shape[0] < self.n_samples_step:
+                time.sleep(0.01)
+                continue
+            if new_data.shape[0] > 2 * self.n_samples_step:
+                print("リアルタイムで処理できる性能が無いため処理ができませんでした")
+                new_data = None
+                keep_data = None
+                continue
+
+        n_samples_new = new_data.shape[0]
+        new_data_transformed = new_data.flatten().astype(np.float32) / 32768.0
+
+        if keep_data is not None:
+            n_samples_take = min([keep_data.shape[0], max(
+                [0, self.n_samples_keep + self.n_samples_length - n_samples_new])])
+            new_data_transformed = np.concatenate(
+                [keep_data[0 - n_samples_take:], new_data_transformed])
+        text = self.w.transcribe(new_data_transformed)
+        print("認識結果:", text)
+        keep_data = new_data_transformed[0 - self.n_samples_keep:].copy()
+        new_data = None
 
-    new_data_transformed = new_data.flatten().astype(np.float32) / 32768.0
+        self.connection.disconnect()
 
-    if keep_data is not None:
-        n_samples_take = min([keep_data.shape[0], max(
-            [0, n_samples_keep + n_samples_length - n_samples_new])])
-        new_data_transformed = np.concatenate(
-            [keep_data[0-n_samples_take:], new_data_transformed])
-    text = w.transcribe(new_data_transformed)
-    print("認識結果:", text)
-    keep_data = new_data_transformed[0-n_samples_keep:].copy()
-    new_data = None
+        while not self.disconnected:
+            time.sleep(0.01)
 
-connection.disconnect()
 
-while not disconnected:
-    time.sleep(0.01)
+if __name__ == "main":
```

Comment:
なぜかこのブロックが実行されないなと思ったら...

```suggestion
if __name__ == "__main__":
```